.github/workflows/integration-test-smoke-suite.yaml

```diff
@@ -71,64 +71,3 @@ jobs:
       if: failure()
       uses: mxschmitt/action-tmate@v3
       timeout-minutes: 60
-
-  TestCephSmokeSuite-Master:
-    if: github.ref == 'refs/heads/master' || github.ref == 'refs/tags/release-*'
-    runs-on: ubuntu-18.04
-    strategy:
-      fail-fast: false
-      matrix:
-        kubernetes-versions : ['v1.15.12','v1.16.15','v1.18.15','v1.19.7','v1.20.2']
```

Comment:
you see `*-Master` should be in integration-test-release.yaml file. So, copied these changes to integration-test-release.yaml file